docs/reference/commandline/inspect.md

```diff
@@ -30,13 +30,26 @@ Options:
 
 ## Description
 
-By default, `docker inspect` will render all results in a JSON array. If the container and
-image have the same name, this will return container JSON for unspecified type.
+Docker inspect provides detailed information on constructs controlled by Docker.
+
+By default, `docker inspect` will render results in a JSON array.
+
+## Request a custom response format (--format)
 If a format is specified, the given template will be executed for each result.
 
 Go's [text/template](http://golang.org/pkg/text/template/) package
 describes all the details of the format.
 
+## Specify target type (--type)
+
+`--type container|image|node|network|service|volume|task|plugin`
+
+It has always been possible for a container to be assigned the same name as an image.
```

Comment:
Would this work (as there's more possibilities than just "images" and "containers" to have the same name);

    The `docker inspect` command matches any type of object by either ID or name.
    In some cases multiple type of objects (for example, a container and a volume)
    exist with the same name, making the result ambigious.

    To restrict `docker inspect` to a specific type of object, use the `--type`
    option.

    The following example inspects a _volume_ named "myvolume"

    ```bash
    $ docker inspect --type=volume myvolume
    ```